### PR TITLE
Don't create PV AMIs in regions that don't support them

### PR DESCRIPTION
**Pull request by bgilbert**
Add `eu-west-3` and `cn-northwest-1`, which don't, to plume.  Add PV AKI ID for `cn-north-1`, which does.

### CODE REVIEW

File: platform/api/aws/images.go [hunk 1]
```diff
@@ -439,6 +454,14 @@ func (a *API) CopyImage(sourceImageID string, regions []string) (map[string]stri
 		return nil, err
 	}
 
+	if *image.VirtualizationType == "paravirtual" {
+		for _, region := range regions {
+			if _, ok := akis[region]; !ok {
```
Comment (euank) [inline platform/api/aws/images.go]: Might as well reuse the `RegionSupportsPV` helper for readability here.

File: platform/api/aws/images.go [hunk 2]
```diff
@@ -439,6 +454,14 @@ func (a *API) CopyImage(sourceImageID string, regions []string) (map[string]stri
 		return nil, err
 	}
 
+	if *image.VirtualizationType == "paravirtual" {
```
Comment (euank) [inline platform/api/aws/images.go]: `ec2.VirtualizationTypeParavirtual` looks like the correct const to use here.

File: cmd/plume/prerelease.go [hunk 3]
```diff
@@ -436,7 +436,9 @@ func awsUploadToPartition(spec *channelSpec, part *awsPartitionSpec, imageName,
 		foundBucketRegion := false
 		for _, region := range part.Regions {
 			if region != part.BucketRegion {
-				destRegions = append(destRegions, region)
+				if !pv || aws.RegionSupportsPV(region) {
+					destRegions = append(destRegions, region)
+				}
```
Comment (euank) [inline cmd/plume/prerelease.go]: Perhaps we should have an "else if pv" that logs to indicate the region pv's being skipped for?

**Comment by bgilbert** [comment]
Updated.

**Pull request merged**